__tests__/app.test.js

```diff
@@ -1,37 +1,45 @@
 const app = require('../app')
-const supertest = require('supertest')
+const request = require('supertest')
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
 
-// beforeAll(done => {
-//     done()
-//   })
 
-// afterAll(done => {
-//     app.close();
-//     done();
-// });
+beforeAll(done => { 
+seed({data})
+    done()
+})
+
```

Comment:
You shouldn't need to be passing `data` inside an object here unless you've modified the seed function.
We're going to want to see `beforeEach` test rather than before all!
We're also going to need to sever the connection to the db `afterAll`
I can see you're utilising the `done` function that jest gives us to inform of the completion of an async process - bear in mind that as `seed` is an asynchronous function, `done` is being invoked before the actual completion of the seed. You'll need to put it's invocation inside a `then` block - but it is also possible to simply `return` the promise and that is enough to tell jest to wait